### DISCUSSION TITLE
[Possibly a bug or unintended] Latex can be loaded in the editor, but cant be viewed from outside

### DISCUSSION THREAD

**Discussion by SPyofcode** [requarks/wiki#6261] [Error / Bug Report]
When I write something like `$\Huge \text{something}$`, it can be loaded in the preview panel

![image](https://user-images.githubusercontent.com/62207095/225388899-fdf6a86e-aca0-4e19-a869-41874774441f.png)

but it should also be loaded on the page.

![image](https://user-images.githubusercontent.com/62207095/225388989-6ba5ccd0-52f6-4476-a129-a739e0ad3444.png)

**Comment by abhinavsood** [comment]
Trying the demo, same issue - I can view the math formulae in Preview Pane but nothing on the published page.

Inspecting the page content, there appears to be an SVG in the DOM - it's just not visible in the browser though.
